### ISSUE TITLE
Certificate with clusters

### ISSUE THREAD

**Issue by hunterboerner** [sasa1977/site_encrypt#61]
How does this library behave when deploying an Elixir cluster behind a load balancer?

**Comment by feld** [comment]
If you're only doing HTTP validation you can cheat the response and allow it to be served from any node. I've done this with Caddy, Nginx, Varnish, etc. And then it helps if you have a way to share the final certificate across all notes (S3, some other shared filesystem, a cache accessible across all Elixir nodes...)

You need to respond to requests at `/.well-known/acme-challenge/TOKEN` with a simple text response of `TOKEN.THUMBPRINT`. All you need to do is share the `THUMBPRINT` value across all nodes. The `THUMBPRINT` is computed from your private key used for ACME (not a TLS private key).

I don't have a simple way to compute that `THUMBPRINT` value in Elixir as I currently use a Bash script with a bunch of functions stolen from another project, but it's certainly possible to do.

**Comment by sasa1977** [comment]
> All you need to do is share the THUMBPRINT value across all nodes.

Right, this is where the problem is. Currently this is not supported. Moreover, it wouldn't work in a split brain scenario, where a node is not visible in the cluster, but it can still serve requests.
